Resources/Locale/en-US/deltav/harpy/singer_system.ftl

```diff
@@ -0,0 +1 @@
+no-sing-while-no-speak = You can not sing right now.
```

Comment:
```suggestion
no-sing-while-no-speak = You can't sing right now.
```

Reply:
You could if you want to, otherwise I'll just mark this as resolved

Reply:
"Can not" isn't consistent with the other localization in the game, so no. Adding an exclamation mark I dont particularly care about, but it should still be cant